src/components/FavoriteServices/DashboardFavorites.tsx

```diff
@@ -0,0 +1,61 @@
+import { Gallery, GalleryItem } from '@patternfly/react-core/dist/dynamic/layouts/Gallery';
+import { Split, SplitItem } from '@patternfly/react-core/dist/dynamic/layouts/Split';
+import { Text, TextContent } from '@patternfly/react-core/dist/dynamic/components/Text';
+import React from 'react';
+
+import useFavoritedServices from '../../hooks/useFavoritedServices';
+import EmptyState from '../FavoriteServices/EmptyState';
+import { bundleMapping } from '../../hooks/useBundle';
+import './DashboardFavorites.scss';
+import { Link } from 'react-router-dom';
+
+import ServiceIcon from './ServiceIcon';
+
+const DashboardFavorites = () => {
+  const favoritedServices = useFavoritedServices();
+
+  const getBundle = (href: string) => bundleMapping[href.split('/')[1]];
+
+  const buildFavorites = () => {
+    return favoritedServices.map((favorite, index) => (
```

Comment:
Store this into a variable no point of having an extra function.

Reply:
Fixed